git2pantheon/api/upload.py

```diff
@@ -95,12 +94,13 @@ def status():
 
     status_message = Status(current_status=status_data['current_status'],
                             file_type=status_data.get('type_uploading', ""),
-                            last_uploaded_file=status_data.get('last_uploaded_file'),
+                            last_uploaded_file=status_data.get('last_file_uploaded'),
```

Comment:
Not sure, if it is a typo as well, rest all looks good.

Reply:
This is a required change. Thanks